hack/ocp-rebase.sh

```diff
@@ -8,18 +8,45 @@
 
 set -e
 
-git remote update
+if ! git remote show | grep -q upstream; then
+    # make sure we have downstream and upstream trees before the script starts
+    if ! git remote add upstream https://github.com/nmstate/kubernetes-nmstate; then
```

Comment:
The script is set -e so there's no need to do this. It will exit if a command fails anyway.